recipes/groot/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+mkdir -p $PREFIX/bin
+mkdir -p src/github.com/will-rowe/groot
+mv cmd src main.go  src/github.com/will-rowe/groot
+cd src/github.com/will-rowe/groot
```

Comment:
Do we need to create a separate directory with
```bash
mkdir -p src/github.com/will-rowe/groot
mv cmd src main.go  src/github.com/will-rowe/groot
cd src/github.com/will-rowe/groot
```
? Is there anything that prevents us from just doing
```bash
#!/bin/bash
mkdir -p $PREFIX/bin

export GOPATH="$SRC_DIR/"
go build -o groot main.go
mv groot $PREFIX/bin
```
?

Reply:
Ah, I see you do
```go
import "github.com/will-rowe/groot/cmd"
```
in https://github.com/will-rowe/groot/blob/0.1/main.go#L23 and the other files.
Okay, then I think
```bash
mkdir -p github.com/will-rowe/groot
mv cmd src main.go github.com/will-rowe/groot
cd github.com/will-rowe/groot
```
or
```bash
mkdir groot
mv cmd src main.go groot/
mkdir -p src/github.com/will-rowe/
mv groot src/github.com/will-rowe/
cd src/github.com/will-rowe/groot
```
should work (the latter if the `src` directory is needed; I have absolutely no experience with Go, so pardon my ignorance).

Reply:
thanks for the help - it was failing because I forgot to run `go get` to get go packages before building (I thought go build ran this for you).